runtimevar/drivertest/drivertest.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018 The Go Cloud Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Package drivertest provides a conformance test for implementations of
+// runtimevar.
+package drivertest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cloud/runtimevar"
+)
+
+// Harness descibes the functionality test harnesses must provide to run
+// conformance tests.
+type Harness interface {
```

Comment:
Why does `*testing.T` need to be passed to each method? Can't it reuse the one given to the `HarnessMaker`?

Reply:
Done.